package.nls.json

```diff
@@ -7,5 +7,6 @@
     "azFunc.startFunctionApp": "Start",
     "azFunc.stopFunctionApp": "Stop",
     "azFunc.restartFunctionApp": "Restart",
-    "azFunc.showExplorerDescription": "Show or hide the Azure Functions Explorer"
+    "azFunc.showExplorerDescription": "Show or hide the Azure Functions Explorer",
+    "azFunc.deployZip": "Deploy as Zip Package"
```

Comment:
How about "Deploy here as zip package"?

Reply:
(but capitalized properly)

Reply:
But 'here' doesn't make sense when the command is run from the command palette.

I created a bug to address the entry points separately: https://github.com/Microsoft/vscode-azurefunctions/issues/15